README.md

```diff
@@ -237,6 +237,7 @@ If you are reading this it looks like you are looking to add an egg to your serv
 * [Subnautica: Nitrox Mod](game_eggs/steamcmd_servers/subnautica_nitrox_mod)
 * [Sven Co-op](game_eggs/steamcmd_servers/svencoop)
 * [The Forest](game_eggs/steamcmd_servers/the_forest)
+* [The Isle](game_eggs/steamcmd_servers/the_isle)
```

Comment:
This should probably have a subsection for `legacy` and `evrima`. Similar to the valheim `vanilla` and `plus mod` layout.

Reply:
Added subsection for Evrima